test/wasmfs/wasmfs_chdir.c

```diff
@@ -85,17 +85,4 @@ int main() {
   ret = getcwd(cwd, sizeof(cwd));
   assert(ret == cwd);
   printf("Current working dir is still: %s\n", cwd);
-
-  // Try to pass a size of 0.
-  errno = 0;
-  getcwd(cwd, 0);
-  printf("Errno: %s\n", strerror(errno));
-  assert(errno == EINVAL);
-
-  // Try to write to a buffer of size 1.
-  errno = 0;
-  char smallBuffer[1];
-  getcwd(smallBuffer, 1);
-  assert(errno == ERANGE);
-  return 0;
```

Comment:
Why remove these? Do we not need this file or parts of it?

Reply:
Sorry, didn't mean to include that hunk.  Fixed.